extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-attachment.js

```diff
@@ -2,6 +2,9 @@ import {expect} from 'chai';
 
 import '../../../amp-story/1.0/amp-story';
 import '../amp-story-shopping';
+
+import {AmpStoryPageAttachment} from 'extensions/amp-story-page-attachment/0.1/amp-story-page-attachment';
```

Comment:
you can register the extension to load in the tests instead, line 15  `extensions: ['amp-story:1.0', 'amp-story-shopping:0.1'],`